src/effects/EffectUI.cpp

```diff
@@ -1158,10 +1138,16 @@ void EffectUIHost::LoadUserPresets()
 
 void EffectUIHost::InitializeRealtime()
 {
-   if (mSupportsRealtime && !mInitialized)
-   {
-      RealtimeEffectManager::Get(*mProject).RealtimeAddEffect(mEffect);
-      
+   if (mSupportsRealtime && !mInitialized) {
+      mpState =
```

Comment:
As I commented earlier, at the moment mpState will be returned as nullptr in the moment I instantiate a BassTreble. 
There is no check here for (mpState == nullptr), I think it should be added